sdk/pinocchio/src/entrypoint/mod.rs

```diff
@@ -109,15 +110,17 @@ const STATIC_ACCOUNT_DATA: usize = size_of::<Account>() + MAX_PERMITTED_DATA_INC
 /// # Important
 ///
 /// The panic handler set up is different depending on whether the `std` library is available to the
-/// linker or not. The `entrypoint` macro will set up a default
-/// panic "hook", that works with the `#[panic_handler]` set by the `std`. Therefore, this macro
-/// should be used when the program or any of its dependencies are dependent on the `std` library.
-///
-/// When the program and all its dependencies are `no_std`, it is necessary to set a
-/// `#[panic_handler]` to handle panics. This is done by the [`crate::nostd_panic_handler`](https://docs.rs/pinocchio/latest/pinocchio/macro.nostd_panic_handler.html)
-/// macro. In this case, it is not possible to use the `entrypoint`
-/// macro. Use the [`crate::program_entrypoint!`] macro instead and set up the allocator and panic
-/// handler manually.
+/// linker or not. The `entrypoint` macro will set up a default panic "hook", that works with the
+/// `#[panic_handler]` set by the `std`. Therefore, this macro should be used when the program or
+/// any of its dependencies are dependent on the `std` library.
+///
+/// When the program and all its dependencies are `"no_std"`, it is necessary to set a
```

Comment:
`no_std` isn't a crate feature, but rather an attribute to be used as `#![no_std]`, like `#[cfg(test)]`, so I would keep it as it was. I don't see it referred to as `"no_std"` anywhere.

The rust embedded book just uses `no_std`: https://docs.rust-embedded.org/book/intro/no-std.html

Reply:
My bad - thank you for checking.
I thought `no_std` is a feature as well.